src/Sentry/Internal/DiscardReason.cs

```diff
@@ -6,6 +6,7 @@ internal record DiscardReason(string Value) : Enumeration(Value)
         public static DiscardReason BeforeSend = new("before_send");
         public static DiscardReason CacheOverflow = new("cache_overflow");
         public static DiscardReason EventProcessor = new("event_processor");
+        public static DiscardReason TransactionProcessor = new("transaction_processor");
```

Comment:
We can't define new discard reasons at this time.  That would require coordination with the backend.  From their perspective, an event discarded from our "transaction processor" would just use `event_processor`, since they don't distinguish between an error event and a transaction event.
https://develop.sentry.dev/sdk/client-reports/#envelope-item-payload